.github/workflows/CI.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x, 20.x]
+    name: Build with ${{ matrix.node-version }}
```

Comment:
Could you change `Build with ${{ matrix.node-version }}` to `Build with Node.js ${{ matrix.node-version }`? Thanks!

Reply:
done